website/docs/docs/cloud/manage-access/scim.md

```diff
@@ -27,32 +41,60 @@ If your IdP isn’t on the list, it can be supported using dbt Cloud APIs (_docs
-6. Navigate to the **General** tab and ensure **Enable SCIM provisioning** is checked or the **Provisioning** tab will not be displayed. 
+1. Log in to your Okta account and locate the app configured for the dbt Cloud SSO integration.
+2. Navigate to the **General** tab and ensure **Enable SCIM provisioning** is checked or the **Provisioning** tab will not be displayed. 
     <Lightbox src="/img/docs/dbt-cloud/access-control/scim-provisioned.png" width="60%" title="Enable SCIM provisioning in Okta." />
-7. Open the **Provisioning** tab and select **Integration**.
-8. Paste the **SCIM base URL** from dbt Cloud to the first field and enter your preferred **Unique identifier field for users** (such as _username_) next.
-9. Click the checkboxes for the following **Supported provisioning actions**:
+3. Open the **Provisioning** tab and select **Integration**.
+4. Paste the **SCIM base URL** from dbt Cloud to the first field and enter your preferred **Unique identifier field for users** (`userName` is recommended) next.
+5. Click the checkboxes for the following **Supported provisioning actions**:
     - Push New Users
     - Push Profile Updates
     - Push Groups
-10. From the **Authentication mode** dropdown, select **HTTP Header**.
-11. In the **Authorization** section, paste the token from dbt Cloud into the **Bearer** field.
+6. From the **Authentication mode** dropdown, select **HTTP Header**.
+7. In the **Authorization** section, paste the token from dbt Cloud into the **Bearer** field.
     <Lightbox src="/img/docs/dbt-cloud/access-control/scim-okta-config.png" width="60%" title="The completed SCIM configuration in the Okta app." />
+8. Ensure that the following provisioning actions are selected:
+    - Create users
+    - Update user attributes
+    - Deactivate users
+    <Lightbox src="/img/docs/dbt-cloud/access-control/provisioning-actions.png" width="60%" title="Ensure the users are properly provisioned with these settings." />
 
-12. Test the connection and click **Save** once completed. 
+9. Test the connection and click **Save** once completed. 
 
 You've now configured SCIM for the Okta SSO integration in dbt Cloud.
+
+### Existing Okta integrations
+
+If you are adding SCIM to an existing Okta integration in dbt Cloud (as opposed to setting up SCIM and Oauth concurrently for the first time), there is some functionality you should be aware of:
+
+- Users and groups already synced to dbt Cloud will become SCIM-managed once the SCIM configuration is completed.
+- You can leverage SCIM to import and manage existing dbt Cloud groups. Update the groups in your IdP with the same naming convention used for dbt Cloud groups. New users, groups, and existing profile changes will be automatically imported into dbt Cloud.
+    - Ensure the **Import users and groups** checkbox is selected in the **Provisioning settings** tab in the Okta SCIM configuration.
```

Comment:
is it import users and groups or import new users and profile updates? not sure if the screenshot in the notion doc is the right one i should be referencing?

Reply:
There are apparently two different okta experiences. The screenshot in the docs is from the less common one but I'll have them doublecheck the language